backend/canisters/user_index/impl/src/model/streak.rs

```diff
@@ -45,6 +45,14 @@ impl Streak {
         }
     }
 
+    pub fn expired_yesterday(&self, now: TimestampMillis) -> bool {
+        if let Some(today) = Streak::timestamp_to_day(now) {
```

Comment:
This will result in loads of unnecessary calls to `timestamp_to_day`.
We could just pass in the day index.

Reply:
Maybe just `expired`? rather than having to be a specific day.